common/buildings/hfe_buildings.txt

```diff
@@ -324,10 +422,19 @@ hfe_building_thoth_module_food = {
 			}
 		}
 	}
+	on_destroy = {
```

Comment:
If it destroyed then it destroyed

Reply:
If you build it and then use the Destroy command, you are dismantling it to be built elsewhere. This is exactly how the Vanilla rendition deals with Fallen Empire Buildings being built outside of a Fallen Empire.